### PR TITLE
icecast: Improve OOB experience - new defaults, add user & group ids

### PR DESCRIPTION
**Pull request by thess**
Make it a little easier to get this up and running - you still need to edit the passwords at a minimum.

@andrenarchy  - please review

Signed-off-by: Ted Hess <thess@kitschensync.net>

### CODE REVIEW

File: multimedia/icecast/files/icecast.init [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+# Copyright (C) 2015 OpenWrt.org
  
 USE_PROCD=1
-START=99
+START=90
 STOP=15
 
 start_service() {
+	user_exists icecast 87 || user_add icecast 87
+	group_exists icecast 87 || group_add icecast 87
```
Comment (andrenarchy) [inline multimedia/icecast/files/icecast.init]: Are these two lines actually necessary? Placing `USERID:=...` should do the job; see [here](https://wiki.openwrt.org/doc/devel/packages). Can you check if user+group are created without these lines? Thx!

Comment (thess) [inline multimedia/icecast/files/icecast.init]: There is a USERID statement in the Makefile too. Adding these lines to the init handles the 'sysupgrade' case when the passwd and group files are preserved.

Comment (andrenarchy) [inline multimedia/icecast/files/icecast.init]: Makes perfect sense. Thx!

**Comment by andrenarchy** [comment]
+1 for merging

**Pull request merged**